QQE-832.md

```diff
@@ -0,0 +1,39 @@
+# QQE-832 Coverage for Quarkus Cache with Infinispan
+
+JIRA link: https://issues.redhat.com/browse/QQE-832
+
+- Quarkus guide: https://quarkus.io/guides/cache-infinispan-reference
+
+## Scope of the testing
+- The test coverage will be verified for environments:  JVM, Native mode and Openshift.
```

Comment:
List of our environments is a matrix with JVM and native on one axe and Baremetal and Openshift on another. Will this coverage be on all four environments?

Reply:
updated